packages/components/src/internal/components/domainproperties/assay/AssayDesignerPanels.tsx

```diff
@@ -116,7 +116,12 @@ export class AssayDesignerPanelsImpl extends React.PureComponent<Props, State> {
                     : textChoiceValidMsg ?? protocolModel.getFirstDomainFieldError();
             const updatedModel = protocolModel.set('exception', exception) as AssayProtocolModel;
             setSubmitting(false, () => {
-                this.setState(() => ({ protocolModel: updatedModel }));
+                this.setState(
+                    () => ({ protocolModel: updatedModel }),
+                    () => {
+                        document.querySelector('#domain-error').scrollIntoView();
```

Comment:
minor: the ID here is unlikely to change, but should it be a constant var? also, thoughts on putting this in a function so that if we decide to change the implementation later there would just be one tough point?

Reply:
Done.